src/page-multi-facility/ReferenceDataModal/SyncNewReferenceData.tsx

```diff
@@ -0,0 +1,104 @@
+import React, { useState } from "react";
+import styled from "styled-components";
+
+import { referenceFacilitiesProp } from "../../database";
+import { useFacilities } from "../../facilities-context";
+import useScenario from "../../scenario-context/useScenario";
+import { Facility, ModelInputs, ReferenceFacility } from "../types";
+import ReferenceDataModal from ".";
+import {
+  getUnmappedFacilities,
+  getUnmappedReferenceFacilities,
+  ReferenceFacilitySelect,
+  ReferenceFacilitySelections,
+  TitleContainer,
+  TitleText,
+} from "./shared";
+
+const Spacer = styled.span`
+  margin-right: 2em;
+`;
+
+const Title: React.FC<Pick<Props, "stateName" | "systemType">> = ({
+  stateName,
+  systemType,
+}) => (
+  <TitleContainer>
+    <TitleText>Prepopulate Data</TitleText>
+    <TitleText>
+      We've found new facility data - select a corresponding facility to
+      autofill with case data
+    </TitleText>
+    <TitleText>
+      State: {stateName || ""}
+      <Spacer />
+      Type of System: {systemType || ""}
```

Comment:
`stateName` and `systemType` are  required, right?  If so, we shouldn't need these `|| ""` fallbacks

Reply:
It looks like stateName is, but the type for `systemType` on the facility is `string | null` so I was wondering if I had to guard for that. When I remove the fallbacks I don't see any TS issues though, so I'll try that!